IdentityCore/src/requests/broker/MSIDSSOExtensionTokenRequestDelegate.m

```diff
@@ -31,19 +31,23 @@ @implementation MSIDSSOExtensionTokenRequestDelegate
 
 - (void)authorizationController:(__unused ASAuthorizationController *)controller didCompleteWithAuthorization:(ASAuthorization *)authorization
 {
+    MSID_LOG_WITH_CTX(MSIDLogLevelInfo, self.context, @"Receive response from SSO extension with authorization is nil: %@", authorization ? @"No" : @"Yes");
+    
     if (!self.completionBlock) return;
     
     NSError *error;
     __auto_type ssoCredential = [self ssoCredentialFromCredential:authorization.credential error:&error];
     if (!ssoCredential)
     {
+        MSID_LOG_WITH_CTX(MSIDLogLevelError, self.context, @"This shouldn't happen: ssoCredential is nill");
```

Comment:
Maybe change to "ssoCredential is nil, error %@" . Same comment to other logs below

Reply:
Good suggestion. Updated